submissions/sprint_a/app.py

```diff
@@ -0,0 +1,14 @@
+from . import app, jsonify
+
+
+@app.route('/')
+def generate_epithets():
+
+    response = {"epithets": []}
+    return jsonify(response)
+
+
+@app.route('/vocabulary')
+def vocabulary():
+    response = {"vocabulary": {}}
+    return jsonify(response)
```

Comment:
You can save yourself a variable assignment and just return the dictionary using jsonify.